packages/venia-ui/lib/components/WishlistPage/wishlistItem.css

```diff
@@ -31,7 +31,13 @@
 .actionsContainer {
     display: flex;
     flex-wrap: wrap;
-    gap: 0.5rem;
+    --gap: 0.5rem;
+    /** polyfill to support flexbox gap in Safari */
+    margin: calc(var(--gap) / -2) calc(var(--gap) / -2);
+}
+
+.actionsContainer > * {
+    margin: calc(var(--gap) / 2) calc(var(--gap) / 2);
```

Comment:
It doesn't block a 👍  but I think this is going to add margin before the first item and after the last item as well, which isn't _exactly_ what `gap` does.

Another option is to go with `grid` instead of `flex`.